custom_test.py

```
@@ -0,0 +1,128 @@
+# Add the src directory to the path
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), "src"))
+
+from mad_spark_alt.core import SmartQADIOrchestrator
+
+
+async def test_with_custom_prompt(
+    problem_statement: str, context: str = "", max_ideas: int = 2
+):
+    """Test the QADI system with a custom prompt"""
+
+    print(f"🎯 Testing with custom prompt:")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)